tests/README.rst

```diff
@@ -0,0 +1,45 @@
+Testing Blivet
+==============
+
+Note: The test suite documented here is available only from the git repository
+not as part of any installable packages.
+
+In order to execute blivet's test suite you need root access on the system.
```

Comment:
This is not really true. Only some tests require root access. Many fairly useful ones don't.